README.rst

```diff
@@ -35,7 +35,7 @@ Required Prerequisites
 ======================
 
 * Python 3.7+
-* cryptography >= 2.5.0
+* cryptography >= 3.4.6
```

Comment:
Is this a breaking change?
What is going to be the version bump we are pushing here?

Reply:
Again, this is on master already: https://github.com/aws/aws-encryption-sdk-python/blob/master/README.rst?plain=1#L38
will fix

Reply:
Done, needed to merge master into `mpl-reviewed` feature branch